tools/export/exporters.py

```diff
@@ -331,7 +331,7 @@ def is_target_supported(cls, target_name):
 
     @classmethod
     def all_supported_targets(cls):
-        return [t for t in TARGET_MAP.keys() if cls.is_target_supported(t)]
+        return [t for t in list(TARGET_MAP) if cls.is_target_supported(t)]
```

Comment:
Should this still be list(TARGET_MAP.keys()) for python2 compatibility?

Just wondering if that works still the same.

Reply:
@teetak01 "list(TARGET_MAP.keys())" and "list(TARGET_MAP.keys())" output is excatly same in both python versions 3.x and 2.7.x

Reply:
Reference if you want to read: https://python-future.org/compatible_idioms.html

Reply:
Looks to fine. Also tested myself a bit between those.